wcivf/apps/elections/templates/elections/includes/_person_card.html

```diff
@@ -35,22 +35,6 @@ <h4 class="candidate_name">
       </p>
       {% endif %}
 
-      <span class="icons">
-      {% if person_post.votes_cast %}
-      {{ person_post.votes_cast|intcomma }} vote{{ person_post.votes_cast|pluralize }}
-      {% endif %}
-
-      {% if person_post.person.statement_to_voters %}<span class="text-icon">Statement</span>{% endif %}
-      {% if person_post.person.pledges.exists %}<span class="text-icon">Pledge</span>{% endif %}
-      {% if person_post.person.cv %}<span class="text-icon">CV</span>{% endif %}
-{#      {% if person_post.person.leaflet_set.exists %}<span class="text-icon">Leaflets</span>{% endif %}#}
-      {% if person_post.person.twitter_username %}<i class="icon-twitter" aria-hidden="true" title="Twitter"></i>{% endif %}
-      {% if person_post.person.facebook_page_url or person_post.person.facebook_personal_url %}<i class="icon-facebook-squared" aria-hidden="true" title="Facebook"></i>{% endif %}
-      {% if person_post.person.linkedin_url %}<i class="icon-linkedin-squared" aria-hidden="true" title="LinkedIn"></i>{% endif %}
-      {% if person_post.person.homepage_url %}<i class="icon-globe" aria-hidden="true" title="Homepage"></i>{% endif %}
-      {% if person_post.person.email %}<i class="icon-mail" aria-hidden="true" title="Email"></i>{% endif %}
-      </span>
```

Comment:
These icons are also used in `_people_list_with_lists.html`.

Also, could you remove the fontello library from the site completely please? I've not done a full investigation, but I know it's included in the scss file and has some files on disk.